src/bindings/python/requirements_test.txt

```diff
@@ -37,4 +37,5 @@ tox
 types-pkg_resources
 wheel>=0.38.1
 protobuf~=3.18.1
-numpy>=1.16.6,<=1.23.4
\ No newline at end of file
+numpy>=1.16.6,<=1.23.4
+torch
```

Comment:
it's not good to have such dependency in python API tests. 
Python API tests test C++ / Python interaction inside OpenVINO. 

About integration of PyTorch and OpenVINO - can / should it be tested via PyTorch Layer tests? 

@jiwaszki what do you think?

Reply:
can do it layer test, but I see onnx dependency there, and decoder is located there, so it seems like a good place for such tests.

Reply:
ONNX will be removed in future, see discussions here https://github.com/openvinotoolkit/openvino/pull/14985

Maybe it's a good time to consider moving Python frontend parts closer to frontends (because when we test frontend, we have to have frameworks, but when we test python API - we definitely don't need frameworks). We have this for C++ where we don't mix code for ngraph, inference and frontends.

Reply:
I agree that makes sense, but I think it should be a different discussion

Reply:
Moved to layer tests for now, but I don't like that solution